etna/transforms/base.py

```diff
@@ -186,6 +188,75 @@ def inverse_transform(self, ts: TSDataset) -> TSDataset:
         :
             TSDataset after applying inverse transformation.
         """
+        pass
+
+
+class IrreversibleTransform(NewTransform):
+    """Base class to create irreversible transforms."""
+
+    def __init__(self, in_column: Union[Literal["all"], List[str], str] = "target"):
+        super().__init__(in_column=in_column)
+
+    def _inverse_transform(self, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Do we need this method here?

Reply:
I guess we do't, define the `inverse_transform` in the base class will be enough.

Reply:
Move it to ReversibleTransform